compose/service.py

```diff
@@ -885,17 +885,19 @@ def has_host_port(binding):
 
         return any(has_host_port(binding) for binding in self.options.get('ports', []))
 
-    def pull(self, ignore_pull_failures=False):
+    def pull(self, ignore_pull_failures=False, silent=False):
```

Comment:
Nevermind then - as long as we have some level of feedback for the user during the operation, that should be sufficient.